setup.cfg

```diff
@@ -0,0 +1,5 @@
+[aliases]
+test=pytest
+
+[pytest]
+addopts=--tb=short -v tests/
```

Comment:
There is much better way to set tests path (http://pytest.org/latest/customize.html#confval-testpaths) but `pytest-runner` is not supporting it, PR is probably coming soon...